bootstrap.php

```diff
@@ -1,9 +1,9 @@
 <?php
-namespace Avatar4eg\UsersList;
+namespace issyrocks12\UsersList;
```

Comment:
Is that change and other similar to that one are intend ? Shouldn't it be reverted ?